src/_guides/libraries/library-tour.md

```diff
@@ -1758,93 +1761,48 @@ Traditionally, AJAX-style apps make heavy use of HttpRequest. Use
 HttpRequest to dynamically load JSON data or any other resource from a
 web server. You can also dynamically send data to a web server.
 
-The following examples assume all resources are served from the same web
-server that hosts the script itself. Due to security restrictions in the
-browser, the HttpRequest class can’t easily use resources that are
-hosted on an origin that is different from the origin of the app. If you
-need to access resources that live on a different web server, you need
-to either use a technique called JSONP or enable CORS headers on the
-remote resources.
```

Comment:
This section is about "Using HTTP resources with HttpRequest", so I've simplified the examples by dropping use of JSON (since JSON is covered in the `dart:convert` section further below in this tour.